components/__tests__/LoadingScreen-test.js

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
```

Comment:
Can we rename the file names to have a . instead of a dash? And make test plural?

So it would be:

```
components/__tests__/LoadingScreen.tests.js
```